debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/BinlogReader.java

```diff
@@ -61,6 +64,7 @@
 
     private static final long INITIAL_POLL_PERIOD_IN_MILLIS = TimeUnit.SECONDS.toMillis(5);
     private static final long MAX_POLL_PERIOD_IN_MILLIS = TimeUnit.HOURS.toMillis(1);
+    private static final Duration HEARTBEAT_INTERVAL = Duration.ofHours(1);
```

Comment:
Now that's something I think should be configurable :) With 0 to turn it off perhaps.

Reply:
Done, configurable as value in seconds